### PR TITLE
docs: update references to tool name and invocation

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: man/oci-image-tool-validate.1.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-% OCI(1) OCI-IMAGE-VALIDATE User Manuals
+% OCI(1) OCI-IMAGE-TOOL User Manuals
 % OCI Community
 % JULY 2016
 # NAME
-oci-image-tool validate \- Validate one or more image files
+oci-image-tool-validate \- Validate one or more image files
```
Comment (cyphar) [inline man/oci-image-tool-validate.1.md]: There should be a space here and in all the other files, since they're subcommands now.

Comment (q384566678) [inline man/oci-image-tool-validate.1.md]: updated.

**Comment by cyphar** [comment]
LGTM. As an aside, I would recommend having more descriptive commit messages. For example, I would name your commit something like `docs: update references to tool name and invocation`.

**Comment by q384566678** [comment]
@cyphar Thanks for your advice, I did not pay attention to these things before.

**Comment by cyphar** [comment]
LGTM. Thanks.

**Comment by coolljt0725** [comment]
LGTM

File: man/oci-image-tool-create.1.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-% OCI(1) OCI-CREATE-RUNTIME-BUNDLE User Manuals
+% OCI(1) OCI-IMAGE-TOOL User Manuals
```
Comment (wking) [inline man/oci-image-tool-create.1.md]: I think we want `OCI-IMAGE-TOOL-CREATE(1) OCI Image Tool User Manuals` here to match the pattern set in Git's `GIT-INIT(1)  Git Manual` (in `git-init(1)`)

Comment (wking) [inline man/oci-image-tool-create.1.md]: Ah, already merged ;).  I'll file a follow-up PR.

**Pull request merged**